e2e/skills/test_skills_crud.py

```
@@ -26,110 +27,438 @@ def test_skills_crud():
+            for i in range(1, 24):
+                print(f"  - skills_{i:02d}_*.png")
 
-            return True
+        except AssertionError as e:
+            print(f"\n[ASSERTION ERROR] {e}")
+            page.screenshot(path="/tmp/skills_error_assertion.png")
+            import traceback
 
+            traceback.print_exc()
+            return False
         except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)